test/mlir/test/gc/gpu-runner/lit.local.cfg

```diff
@@ -1,2 +1,2 @@
-if not config.gc_use_gpu:
+if not config.gc_use_imex:
```

Comment:
Similar to above... Do we need to distinguish general GPU tests from IMEX tests?